pinot-common/src/main/java/org/apache/pinot/common/utils/PinotDataType.java

```diff
@@ -60,6 +63,16 @@ public double toDouble(Object value) {
       return ((Boolean) value) ? 1d : 0d;
     }
 
+    @Override
+    public boolean toBoolean(Object value) {
```

Comment:
shall this return type be `Boolean` to allow null value in future?

Reply:
We won't allow `null` value inside this type conversion util class. `null` value should be handled outside.